lib/bot/events/temp_voice_event_container.rb

```diff
@@ -31,9 +572,15 @@ def self.handle_channel_join(event)
     discord_jump_channel =
       server.voice_channels.detect { |c| c.id == channel.id }
 
+    # Get adjective
+    adjective = adjective_list[rand(adjective_list.length)]
+
+    # Get noun
+    noun = noun_list[rand(noun_list.length)]
```

Comment:
Magic things that you can do with `ruby`:
```suggestion
    noun = noun_list.sample
```